Q/QuantRegBuilder/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QuantRegBuilder"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fogarty-ben/QuantReg.jl-Builder.git", "486b3d9ce5f5a0b3c1172c77a47166c57f1cd501")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd QuantReg.jl-Builder/
+mkdir -p ${libdir}
+gfortran -fPIC -shared -std=legacy rqbr.f -o ${libdir}/rqbr.${dlext} 
+gfortran -fPIC -shared -std=legacy rqfnb.f -o ${libdir}/rqfnb.${dlext} 
+install_license /usr/share/licenses/GPL3
```

Comment:
You're free to use whatever license you want for your own code, just note that most julia packages use MIT license. Also, it would be better to include the license file directly in the upstream repository :wink:

Reply:
Ahh, this makes perfect sense; the combined sum of this project is actually porting over some R code that is under the GPL3 license hence that choice. Making the change to storing the license in the upstream repository right now. Thanks again!